proof_ipa.go

```diff
@@ -145,7 +145,7 @@ func getProofElementsFromTree(preroot, postroot VerkleNode, keys [][]byte, resol
 		// keys were sorted already in the above GetcommitmentsForMultiproof.
 		// Set the post values, if they are untouched, leave them `nil`
 		for i := range keys {
-			val, err := postroot.Get(keys[i], resolver)
+			val, err := postroot.Get(keys[i], 0, resolver)
```

Comment:
I don't think this is correct: if the epoch is 0 here, then you would get `errEpochExpired` all the time, wouldn't you?